### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by schlotze**
Please check the updated Readme.
Specifically on how to describe the "build" and "run" part I am not sure.
@philipreimer Maybe you could add here some sentences.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,254 @@
 # smartclide-context
 SmartCLIDE Context Handling Component
+
+## Preconditions to build and run Context Handling
+
+To build and run Context Handling, the following software is required:#
+
+- Java (at least version 11)
+- Apache Maven (min V3.6)
```
Comment (philipreimer) [inline README.md:9]:
```suggestion
- Apache Maven (at least version 3.5.4)
```

according to parent pom

File: README.md [hunk 2]
```diff
@@ -1,2 +1,254 @@
...
+- Docker [Desktop] (for running Context Handling)
```
Comment (philipreimer) [inline README.md:10]: Is docker really needed to run context-handling?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,254 @@
...
+
+Note: Currently the configuration files are packed into the container images, which means that on each configuration change a new container has to be created and deployed.
+
+## How to run Context Handling
+
+There are two possibilites to execute Context Handling. 
+
+1. Building and runnning the locally built Context Handling can be done using the following command:
+   ```
+   buildAndRunCHLocally.sh
```
Comment (philipreimer) [inline README.md:30]: This file does not seem to exist in the branch. Did you maybe forget to commit it?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,254 @@
...
+## How to configure Context Handling
+
+The Context Handling configuration files are store in the following folder:
+`./resources/`
+
+### Monitoring Config
+
+In the following the xml schema for the monitoring configuration is listed. Below the xsd the configuration elements are described in detail.
+
+**monitoring-config.xsd**
```
Comment (philipreimer) [inline README.md:47]: I would suggest to just link to the XSD files from here. That way the README content cannot get out of sync with the actual XSD content.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,254 @@
...
+        </interpreter>
+    </interpreters>
+
+    <monitors>
+        <monitor id="monitor-git" datasource="datasource-git" interpreter="interpreter-git" index="index-git"/>
+    </monitors>
+</config>
+```
+
+The "monitoring-config.xml" configuration file can also be changed by using the Admin UI.
```
Comment (philipreimer) [inline README.md:165]: Some more information how to access the Admin UI would be good.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,254 @@
...
+Each datasource entry has the following mandatory attributes
+
+- id:The unique name of the datasource
+- type:The type of the datasource. Possible values are: file, webservice, database
+- monitor:The class of the monitor to be used. Possible values are:
+    - package de.atb.context.monitoring.monitors.database.DatabaseMonitor
+    - package de.atb.context.monitoring.monitors.file.FileSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FilePairSystemMonitor
+    - package de.atb.context.monitoring.monitors.file.FileTripletSystemMonitor
+    - package de.atb.context.monitoring.monitors.webservice.WebServiceMonitor
```
Comment (philipreimer) [inline README.md:188]: The monitors / data source implementations added by SmartCLIDE should be listed as well.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,254 @@
...
+Each interpreter entry has the following mandatory attributes
+
+- id: The unique name of the interpreter
+- configuration
+    - analyser: The analyser class to be used. The following implementations are available:
+        - package de.atb.context.monitoring.analyser.database.DatabaseAnalyser
+        - package de.atb.context.monitoring.analyser.file.FileAnalyser
+        - package de.atb.context.monitoring.analyser.file.FilePairAnalyser
+        - package de.atb.context.monitoring.analyser.file.FileTripletAnalyser
+        - package de.atb.context.monitoring.analyser.webservice.WebServiceAnalyser
```
Comment (philipreimer) [inline README.md:209]: The analysers / parsers added by SmartCLIDE should be listed as well.

**Review by philipreimer — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (schlotze) [reply README.md:10]: No it is not. Only required for deploing it on our Cluster

File: README.md (hunk 3 above)
Comment (schlotze) [reply README.md:30]: No I was wondering if it makes sense to have such a script or whether this is too much. What do you think?

File: README.md (hunk 5 above)
Comment (schlotze) [reply README.md:165]: I think this should be remove for now, because the UI needs major changes

File: README.md (hunk 4 above)
Comment (schlotze) [reply README.md:47]: [x] done

File: README.md (hunk 6 above)
Comment (schlotze) [reply README.md:76]: [x] done

File: README.md (hunk 7 above)
Comment (schlotze) [reply README.md:102]: [x] done

File: README.md (hunk 3 above)
Comment (philipreimer) [reply README.md]: removed it

**Review by philipreimer — DISMISSED** [review]

File: README.md [hunk 8]
```diff
@@ -1,2 +1,112 @@
...
+        - package de.atb.context.monitoring.parser.file.FileParser
+        - package de.atb.context.monitoring.parser.file.FilePairParser
+        - package de.atb.context.monitoring.parser.file.FileTripletParser
+        - package de.atb.context.monitoring.parser.webservice.MessageBrokerParser
+        - package de.atb.context.monitoring.parser.webservice.WebServiceParser
+        - package de.atb.context.monitoring.parser.GitlabCommitParser
+        - package de.atb.context.monitoring.parser.GitParser
+
+    - type: Currently only used for File analyser and parser. Defines the file extensions to be used.
+
```
Comment (philipreimer) [inline README.md:111]: @schlotze should there be some documentation for the `<monitors>` section?

File: README.md [hunk 9]
```diff
@@ -1,2 +1,112 @@
...
+- id: The unique name of the index
+- location: The URI of the location the index is stored
+
+#### datasources
+
+Each datasource entry has the following mandatory attributes
+
+- id:The unique name of the datasource
+- type:The type of the datasource. Possible values are: file, webservice, database
```
Comment (philipreimer) [inline README.md:64]: Aren't there more possible data sources by now?

Comment (schlotze) [reply README.md]: updated

**Review by philipreimer — APPROVED** [review]

**Review by philipreimer — APPROVED** [review]

**Pull request merged**